app/controllers/api_controller/generic.rb

```diff
@@ -132,6 +132,19 @@ def set_ownership_resource(type, id, data = nil)
       end
     end
 
+    def refresh_dialog_fields_action(dialog, refresh_fields, resource_ident)
+      result = {}
+      refresh_fields.each do |field|
+        dynamic_field = dialog.field(field)
+        return action_result(false, "Unknown dialog field #{field} specified") unless dynamic_field
+        unless dynamic_field.respond_to?(:update_and_serialize_values)
```

Comment:
I believe non dynamic dialog fields, though it might be safe to remove this tests as the method is in  the dialog field base class, It just hasn't been tested as such though (i.e. 400 vs 500).